### DISCUSSION TITLE
The concentration of intravenous bolus

### DISCUSSION THREAD

**Discussion by Supercn01** [Open-Systems-Pharmacology/Forum#1681] [Models]
Hi everyone.
    I have simulate a drug with intravenous bolus administarion,but I wonder why the concentraion of the first administation is higher than next. I will very appreciate if anyone could tell me the reason!
![264d9bea7d1cef35c398453ae986066](https://github.com/Open-Systems-Pharmacology/Forum/assets/85626813/8182b70c-44b5-4ba5-9b39-4cbd188f57ea)

**Comment by HenrikCordes** [comment]
Hi @Supercn01,

I is rather challenging to say without having the simulation file nor knowing your model configuration and parameter. 
Possibility the resolution (time steps) of your simulation are different over the time profile. 
The default is that the first hour is rather fine grained and the later time frames rather rough. 

Try using the same simulation resolution for the whole time frame.

Best
Henrik
